tests/Tests.cs

```diff
@@ -25,11 +25,13 @@ public class ControllerTests
 
 			// assert
 			var viewResult = Assert.IsType<ViewResult>(result);
-			Assert.Equal(expected: expectedDefaultUrl, actual: viewResult.ViewData["url"] as String);
-			Assert.Equal(expected: expectedDefaultType, actual: viewResult.ViewData["type"] as String);
-			Assert.Equal(expected: expectedDefaultTitle, actual: viewResult.ViewData["title"] as String);
-			Assert.Equal(expected: expectedDefaultDescription, actual: viewResult.ViewData["description"] as String);
-			Assert.Equal(expected: expectedDefaultImage, actual: viewResult.ViewData["image"] as String);
+			var viewModel = viewResult.Model as MetaViewModel;
```

Comment:
Style: This should be an `Assert.IsType<MetaViewModel>(viewResult.Model)` so if the test fails, there is a very clear error indicating that the returned value isn't the right type.  By using `as MetaViewModel`, the error will be a Null Reference Exception instead of an invalid type exception, which is much less clear.